main.go

```diff
@@ -153,11 +153,20 @@ func startLeaderElection(leaderElectionClient kubernetes.Interface, recorder rec
 	if err != nil {
 		klog.Fatalf("error getting hostname: %s", err.Error())
 	}
+	
+	kubeconfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{},
+	)
+	namespace, _, err := kubeconfig.Namespace()
+	if err != nil {
+		klog.Fatalf("error getting namespace: %s", err.Error())
```

Comment:
Ok, I saw the comments on the open issue, and I understand you use case. For backwards compatibility instead of failing hard here, could you please make it fall back to the submariner namespace? ej:

if err != nil {
    namespace = "submariner"
}

Reply:
@negashev ping ^